src/main/resources/data-migration/example_mapped_language_data.csv

```diff
@@ -0,0 +1,4 @@
+    reference     ,      state       ,      event_id      , case_data_id ,submitted_language                                      , existing_language_value                        ,interpreter,mapped_language_value,,Uma Comments
```

Comment:
do we need the event id in csv ?

Reply:
not really, it was just there in the original mapping sheet. I'll remove it
in reality we just need case ref, interpreter, existing language and mapped language
I'll remove all the other columns

Reply:
will remove the example file completely, and just mention the required columns in the instructions